contracts/src/TokenSpoke/ERC20TokenSpoke.sol

```diff
@@ -171,7 +171,7 @@ contract ERC20TokenSpoke is IERC20TokenBridge, ERC20, TokenSpoke {
     function _handleFees(
         address feeTokenAddress,
         uint256 feeAmount
-    ) internal override returns (uint256) {
```

Comment:
What's the motivation to mark these virtual?

Reply:
Similar to the other internal spoke functions, to allow developers to provide their own spoke implementations and override these default implementations when needed.